packages/widget/src/hooks/use-swap-widget.ts

```diff
@@ -70,24 +70,14 @@ export function useSwapWidget(persistSwapWidgetState = true) {
   const destinationChainIDs = filter?.destination
     ? Object.keys(filter.destination)
     : undefined;
-  const allChainIDs = useMemo(() => {
-    return [...(sourceChainIDs ?? []), ...(destinationChainIDs ?? [])];
-  }, [sourceChainIDs, destinationChainIDs]);
 
   const {
     assetsByChainID,
     getFeeAsset,
     isReady: isAssetsReady,
     getAsset,
   } = useAssets();
-  const { data: chains } = useChains({
-    select: (chains) => {
-      if (allChainIDs.length === 0) return chains;
-      return chains?.filter(({ chainID }) => {
-        return allChainIDs.includes(chainID);
-      });
-    },
-  });
```

Comment:
if either filter.source or filter.destination is undefined we will effectively filter all of those chains out, and it's not super feasible to simply include "all of the chainIDS" when filter.source/filter.destination is undefined, let's simplify this to simply not select any chains and always return all of the chains